debugger/package-lock.json

```diff
@@ -0,0 +1,13599 @@
+{
```

Comment:
I think you can zip this file as it contains some internal links, and yarn.lock should be sufficient

Reply:
removed